manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml

```diff
@@ -0,0 +1,44 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Waterfox.Waterfox
+PackageVersion: G6.0.3
+PackageLocale: en-US
+Publisher: WaterfoxLimited
```

Comment:
This should be BrowserWorks

Reply:
Unless the publisher is also listed as BrowserWorks in the control panel, this will not be changed.